packages/parcel-transformer/src/transformer.tsx

```diff
@@ -49,13 +48,18 @@ export default new Transformer<UserlandOpts>({
       return undefined;
     }
 
-    const ast = await parseAsync(code, {
+    const program = await parseAsync(code, {
       filename: asset.filePath,
       caller: { name: 'compiled' },
       rootMode: 'upward-optional',
     });
 
-    return ast;
+    return {
+      type: 'babel',
+      version: '7.0.0',
+      code,
```

Comment:
You should remove `code`, as it's not used elsewhere in the transformer and shouldn't be needed in general